### PR TITLE
🔥  remove imageRelPath

### PR DESCRIPTION
**Pull request by kirrg001**
refs #7488

- instead add a static image prefix to the url helper

### CODE REVIEW

File: core/server/utils/url.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ function urlFor(context, data, absolute) {
             secure = data.author.secure;
         } else if (context === 'image' && data.image) {
             urlPath = data.image;
-            imagePathRe = new RegExp('^' + getSubdir() + '/' + config.get('paths').imagesRelPath);
+            imagePathRe = new RegExp('^' + getSubdir() + '/' + module.exports.staticImageUrlPrefix);
```
Comment (ErisDS) [inline core/server/utils/url.js]: This looks super duper weird to me?!

File: core/server/config/utils.js [hunk 2]
```diff
@@ -44,10 +43,9 @@ exports.makePathsAbsolute = function makePathsAbsolute(obj, parent) {
         } else {
             if (_.isString(configValue) &&
                 (configValue.match(/\/+|\\+/) || configValue === '.') &&
-                (configValue[0] !== '/' && configValue[0] !== '\\') &&
-                pathsKey !== 'imagesRelPath'
+                (configValue[0] !== '/' && configValue[0] !== '\\')
             ) {
-                self.set(parent + ':' + pathsKey, path.join(__dirname + '/../../../', configValue));
+                self.set(parent + ':' + pathsKey, path.join(process.cwd(), configValue));
```
Comment (ErisDS) [inline core/server/config/utils.js]: Struggling to reason about this change. Are there tests for this?

Comment (kirrg001) [inline core/server/config/utils.js]: sorry, i will remove this change as it has nothing todo with this PR 👍

File: core/server/utils/url.js (hunk 1 above)
Comment (kirrg001) [inline core/server/utils/url.js]: this is weird!
we could add a var on top and use it for export and internal file usage?

Comment (ErisDS) [inline core/server/utils/url.js]: yar I think so 👍

**Comment by ErisDS** [comment]
Need a rebase, but GTG!

**Pull request merged**